builtin/client/init.lua

```diff
@@ -21,3 +21,7 @@ core.register_on_formspec_input(function(formname, fields)
 		core.send_respawn()
 	end
 end)
+
+core.register_on_mods_loaded(function()
+	core.log("Client mods loaded.")
+end)
```

Comment:
Same here as with the SSM builtin part. It however might fit better into the preview CSM mod.